pkgs/servers/sql/postgresql/ext/citus.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, postgresql, curl, lz4 }:
+
+stdenv.mkDerivation rec {
+  name = "citus-${version}";
+  version = "11.0.6";
+
+  src = fetchFromGitHub {
+    owner  = "citusdata";
+    repo   = "citus";
+    rev    = "refs/tags/v${version}";
+    sha256 = "zBygZLaFUnAkDEgrdeGwnWl1y+gIK4JWmB7gW482xx8=";
+  };
+
+  buildInputs = [ postgresql curl lz4 ];
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  installPhase = ''
+    mkdir -p $out
+  '';
+
+  passthru = {
+    versionCheck = postgresql.compareVersion "11" >= 0 && postgresql.compareVersion "14" < 0;
+  };
+
+  meta = with lib; {
+    description = "Transparent, distributed sharding and replication for PostgreSQL";
+    homepage    = https://www.citusdata.com/;
+    maintainers = with maintainers; [ ];
```

Comment:
[docs say](https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-packages) "Maintainers must be set"